File: C/practicals/Practical-3.c/p1.c
#include<stdio.h>

int main(){

	int num;
	printf("Enter your number: ");
	scanf("%d",&num);

	if(1<=num && num<=1000){
		printf("%d is in the range of 1 to 1000 \n",num);
	}else if (num>1000){
		printf("%d is not in the range of 1 to 1000 \n",num);
	}
}

File: JAVA/AsgnPract/4/p1.java
import java.util.*;

class one{


	public static void main(String[] args){


		Scanner sc = new Scanner(System.in);

		System.out.print("Enter num you wanna check: ");

		int num = sc.nextInt();

		System.out.println();

		if(1<= num  && num<= 100){

			System.out.println(num +" is in the range of 1 to 100");
			System.out.println();

		}else {

			System.out.println(num + " is not int the range 1 to 100");
			System.out.println();
		}
	}


}
